apps/api/package.json

```diff
@@ -30,11 +30,11 @@
 		"@types/body-parser": "^1.19.5",
 		"body-parser": "^1.20.3",
 		"cors": "^2.8.5",
-		"drizzle-orm": "^0.38.3",
+		"drizzle-orm": "^0.39.1",
```

Comment:
See note in `service/collaboratorsService` https://github.com/Pan-Canadian-Genome-Library/daco/pull/175/files#diff-fe99e87f7017cc3474b7915429480c6fb9eb76d78603256f7645fa59ce2a2d6e

I updated this to attempt fixing a bug (bug persists), could be removed if there's any issue